DockerfileTomcatDCEVM

```diff
@@ -0,0 +1,26 @@
+ARG VERSION
+
+FROM silvermedia/tomcat:$VERSION
+
+RUN rm -rf /usr/local/openjdk-11
+
+RUN apt-get update && \
+    apt-get install -y curl && \
+    apt-get clean
+
+RUN curl -L -o /tmp/dcevm.tar.gz https://github.com/TravaOpenJDK/trava-jdk-11-dcevm/releases/download/dcevm-11.0.5%2B5/java11-openjdk-dcevm-linux.tar.gz
+
+RUN mkdir /usr/local/dcevm && \
+    tar -xf /tmp/dcevm.tar.gz -C /usr/local/dcevm --strip 1 && \
+    rm /tmp/dcevm.tar.gz
```

Comment:
This will create layer which will contain `/tmp/dcevm.tar.gz`, and removal in the next layer wont actually remove the data. You need to merge these two commands.